torrentfs/torrent.go

```diff
@@ -204,6 +204,10 @@ func (t *Torrent) ReloadTorrent(data []byte, tm *TorrentManager) error {
 var maxCited int64 = 1
 
 func (t *Torrent) IsAvailable() bool {
+	if t.InfoHash() == "de58609743e5cd0cb18798d91a196f418ac25016" {
+		return true
+	}
+
```

Comment:
You can upload the correct file, then you only need to hack the hash code in synapse/*_infer.go

Reply:
File storage has been moved here https://github.com/CortexFoundation/torrentfs